.github/workflows/linux-cpu-arm64-build.yml

```diff
@@ -7,42 +7,48 @@ env:
   ort_dir: "onnxruntime-linux-aarch64-1.16.3"
   ort_zip: "onnxruntime-linux-aarch64-1.16.3.tgz"
   ort_url: "https://github.com/microsoft/onnxruntime/releases/download/v1.16.3/onnxruntime-linux-aarch64-1.16.3.tgz"
+  docker_image: "onnxruntimecpubuild"
 jobs:
   job:
+    name: "Linux CPU ARM64 Build"
     runs-on: [ "self-hosted", "1ES.Pool=onnxruntime-genai-Ubuntu2004-ARM-CPU" ]
     steps:
       - name: Checkout OnnxRuntime GenAI repo
-        uses: actions/checkout@v2
-
-
+        uses: actions/checkout@v4
+        with:
+          submodules: true
 
       - name: Download OnnxRuntime
         run: |
+          echo "Downloading ${{ env.ort_url }}"
           curl -L -o ${{ env.ort_zip }} ${{ env.ort_url }} 
-
-      - name: Unzip OnnxRuntime
-        run: |
+          echo "Unzipping ${{ env.ort_zip }}"
           tar -xzf ${{ env.ort_zip }}
           rm ${{ env.ort_zip }}
-
-      - name: Rename OnnxRuntime to ort
-        run: |
+          echo "Renaming ${{ env.ort_dir }} to ort"
           mv ${{ env.ort_dir }} ort
 
-      - name: Git Submodule Update
+      - name: Replace vcpkg linux.cmake aarch64-linux-gnu-gcc with gcc
         run: |
```

Comment:
Set CC and CXX environment variables instead. 
Like:
export CC=gcc
export CXX=g++

Reply:
Tried, but does not work.

Reply:
```
elseif(VCPKG_TARGET_ARCHITECTURE STREQUAL "arm64")
    set(CMAKE_SYSTEM_PROCESSOR aarch64 CACHE STRING "")
    if(CMAKE_HOST_SYSTEM_NAME STREQUAL "Linux")
        if(NOT DEFINED CMAKE_CXX_COMPILER)
            set(CMAKE_CXX_COMPILER "aarch64-linux-gnu-g++")
        endif()
        if(NOT DEFINED CMAKE_C_COMPILER)
            set(CMAKE_C_COMPILER "aarch64-linux-gnu-gcc")
        endif()
        if(NOT DEFINED CMAKE_ASM_COMPILER)
            set(CMAKE_ASM_COMPILER "aarch64-linux-gnu-gcc")
        endif()
        if(NOT DEFINED CMAKE_ASM-ATT_COMPILER)
            set(CMAKE_ASM-ATT_COMPILER "aarch64-linux-gnu-as")
        endif()
```

Reply:
These settings are for cross-compiling. It means vcpkg probably doesn't support build natively on arm64.

Reply:
So, should we use it?

Reply:
I don't know. Let me search around.